### ISSUE TITLE
No announcement for the slides on the carousel

### ISSUE THREAD

**Issue by nemain00** [ALASdoo/alasdoo_com_v2#89]
When focusing on the slide of the carousel, the screen reader does not announce any information for the focused slide.
The screen reader announces: 
"Go to slide  'number of the slide'  button" - when tabbing and clicking on the form field key
"grouping  Quation mark  graphic  " - when clicking on the Go to next graphic key

**Comment by Tajana-Somogyi** [comment]
@a-kosanovic Changes needed:

- [ ] Do not change automatically, only when user manually activates (by default it is set on the first slide) 
- [ ] Associate content with commands (radio button)
- [ ] See carousel best practices in issue #48

**Comment by Tajana-Somogyi** [comment]
We decided that we'll chose 3 most important testimonials instead of the carousel.
